firefox-ios/firefox-ios-tests/Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -655,8 +655,8 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
             }
         }
         screenState.gesture(forAction: Action.FxATypePasswordNewAccount) { userState in
-            app.secureTextFields.element(boundBy: 1).tap()
-            app.secureTextFields.element(boundBy: 1).typeText(userState.fxaPassword!)
+            app.secureTextFields.element(boundBy: 0).tap()
```

Comment:
did not we change this recently? I think this is chaging from different versions and so the tests pass in one but fail in another... 
Let's make sure where the changes apply (release, v124, 125...)

Reply:
these tests started to fail in every release version, i am guessing whenever the change was made it was backported

Reply:
yes it would, i first notice these fails on 125 having the same problems

Reply:
Let me check with the SyncIntegrationTests. I made this change because the sign-in page for existing accounts is different from new account's.